### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by Aidanjm13** [psu-edu/agile_stem_basic_game#125]
A settings system and page should be added to the game that allows for better accessibility and personalization for the player. Some examples may be volume, sensitivity, controls, language, etc.
